stake-pool/cli/src/main.rs

```diff
@@ -491,6 +491,59 @@ fn command_create_pool(
     Ok(())
 }
 
+fn create_token_metadata(
+    config: &Config,
+    stake_pool_address: &Pubkey,
+    name: String, 
+    symbol:String, 
+    uri:String) -> CommandResult {
+    let stake_pool = get_stake_pool(&config.rpc_client, stake_pool_address)?;
+
+    let mut signers = vec![config.fee_payer.as_ref(), config.staker.as_ref()];
```

Comment:
The manager should be the one to sign this
```suggestion
    let mut signers = vec![config.fee_payer.as_ref(), config.manager.as_ref()];
```